File: iron_worker_examples/node/worker101/enqueue.js
var https = require("https");

function queue_task(project, token, code_name,query) {
    // Build the payload
    var payload = {
        "query": query
    };

    var req_json = {
        "tasks": [{
            "code_name": code_name,
            "payload": JSON.stringify(payload)
        }]
    }

    // Convert the JSON data
    var req_data = JSON.stringify(req_json);

    // Create the request headers
    var headers = {
        'Authorization': 'OAuth ' + token,
        'Content-Type': "application/json"
    };

    // Build config object for https.request
    var endpoint = {
        "host": "worker-aws-us-east-1.iron.io",
        "port": 443,
        "path": "/2/projects/" + project + "/tasks",
        "method": "POST",
        "headers": headers
    };

    var post_req = https.request(endpoint, function(res) {
        console.log("statusCode: ", res.statusCode);

        res.on('data', function(d) {
            process.stdout.write(d);
        });
    });

    post_req.write(req_data)
    post_req.end();

    post_req.on('error', function(e) {
        console.error(e);
    });
}

queue_task("YOUR_PROJECT_ID", "YOURTOKEN", "NodeWorker101","Heyyaa");

File: iron_worker_examples/dotnet/worker101/enqueue.cs
using System;
using System.IO;
using System.Net;
using System.Collections;
using System.Collections.Generic;
using System.Text;
using System.Web;

abstract
public class EnQueueTask
{
	static public void Main (string[] args)
	{
string project_id = "YOUR_PROJECT_ID";
string token = "YOUR_TOKEN";
string url = "http://worker-aws-us-east-1.iron.io/2/projects/" + project_id + "/tasks";
HttpWebRequest req = WebRequest.Create(new Uri(url))
                     as HttpWebRequest;
req.Method = "POST";
req.ContentType = "application/json";
//setting oauth authorization
req.Headers.Add("Authorization", "OAuth " + token);
string paramz = "{\"tasks\":[{\"code_name\":\"MonoWorker101\",\"payload\":\"{'query':'Heyyaa'}\"},\"delay\":\"5\"]}";

// Encode the parameters as form data:
byte[] formData =
    UTF8Encoding.UTF8.GetBytes(paramz);
req.ContentLength = formData.Length;

// Send the request:
using (Stream post = req.GetRequestStream())
{
  post.Write(formData, 0, formData.Length);
}

// Pick up the response:
string result = null;
using (HttpWebResponse resp = req.GetResponse()
                              as HttpWebResponse)
{
  StreamReader reader =
      new StreamReader(resp.GetResponseStream());
  result = reader.ReadToEnd();
}
}}